asm/custom-functions/src/lib.rs

```diff
@@ -367,6 +401,22 @@ fn rando_text_command_handler(_event_flow_mgr: *mut c_void, p_flow_element: *con
                 UNOBTAINED_TEXT.as_ptr()
             };
             text_manager_set_string_arg(life_tree_fruit_text as *const c_void, 2);
+
+            // Tadtones obtained.
+            text_manager_set_num_args(&[storyflag_get_to_value(953) as u32]);
+        }
+        74 => { // Increment storyflag counter
+            let flag = flow_element.param1;
+            let increment = flow_element.param2;
+            
+            storyflag_set_to_value(flag, storyflag_get_to_value(flag) + increment);
+        }
+        75 => { // Have collected all tadtone groups?
+            let tadtone_groups_left = 17 - storyflag_get_to_value(953);
```

Comment:
```suggestion
            let tadtone_groups_left: u32 = 17.saturating_sub(storyflag_get_to_value(953)).into();
```
Just to be safe, to not wrap around (side note, this only wraps around in release mode, in debug mode it would panic instead)
Also you can just use `into` here to get rid of the casts later, by just turning the u16 into u32 here